packages/terra-switch/src/terra-dev-site/doc/switch/Switch.1.doc.mdx

```diff
@@ -0,0 +1,60 @@
+import { Badge } from 'terra-switch/package.json?dev-site-package';
+
+import DefaultSwitch from '../example/DefaultSwitch?dev-site-example';
+import SwitchWithOnChange from '../example/SwitchWithOnChange?dev-site-example';
+import DisabledSwitch from '../example/DisabledSwitch?dev-site-example';
+import SwitchWithLongLabelText from '../example/SwitchWithLongLabelText?dev-site-example';
+
+import SwitchPropsTable from 'terra-switch/src/Switch?dev-site-props-table';
+
+<Badge />
+
+# Terra Switch
+
+The terra-switch component are like their real-world counterparts, are used to indicate whether a setting is on or off and provide users with instantaneous feedback. 
```

Comment:
```suggestion
The terra-switch component is, like its real-world counterpart, is used to indicate whether a setting is on or off and provide users with instantaneous feedback. 
```

Reply:
Lets keep the documentation/description of terra-switch consistent by making the same change in the "README.md" file.

Reply:
updated with commit : d8ea53ba069375ab4efb9abebf1bd67951aa067f